csi/main.py

```diff
@@ -1,22 +1,41 @@
 """
 Starting point of CSI driver GRP server
 """
-from concurrent import futures
-import time
 import logging
 import os
-
-import grpc
+import time
+from concurrent import futures
 
 import csi_pb2_grpc
-from identityserver import IdentityServer
+import grpc
 from controllerserver import ControllerServer
+from identityserver import IdentityServer
+from kadalulib import CommandException, logging_setup
 from nodeserver import NodeServer
-from kadalulib import logging_setup
-
+from volumeutils import (HOSTVOL_MOUNTDIR, get_pv_hosting_volumes,
+                         mount_glusterfs)
 
 _ONE_DAY_IN_SECONDS = 60 * 60 * 24
 
+def mount_storage():
+    """
+    Mount storage if any volumes exist after a pod reboot
+    """
+    if os.environ.get("CSI_ROLE", "-") != "provisioner":
+        logging.debug("Volume need to be mounted on only provisioner pod")
+        return
+
+    host_volumes = get_pv_hosting_volumes({})
+    for volume in host_volumes:
+        hvol = volume['name']
+        mntdir = os.path.join(HOSTVOL_MOUNTDIR, hvol)
+        try:
+            mount_glusterfs(volume, mntdir)
+        except CommandException:
+            logging.error("Unable to mount volume %s" % hvol)
+        logging.info("%s is mounted successfully" % hvol)
```

Comment:
looking at the pylint error, and to keep consistency with other places, best is to do it as below:

```suggestion
        logging.info(logf("Mounted successfully" , hvol=hvol))
```

same for above. (note the required `from kadalulib import logf`)

Reply:
Sure, will take care before that I've a query and will state in the comment.